src/tests/trip/trip-request.test.js

```diff
@@ -0,0 +1,81 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../../app';
+import { loggedInToken, userWithNoTripToken } from '../fixtures/users.fixture';
+
+chai.should();
+chai.use(chaiHttp);
+
+describe('/Get view requested trips ', () => {
+  it('App should show requested trip', (done) => {
```

Comment:
May you please a before hook to clean db in order to avoid conflicts with future tests with other people.(It has been happening earlier).